als-structure/shared/src/test/resources/Async20/declarations/messages/api-yaml-outline.json

```diff
@@ -81,7 +81,7 @@
                 "range": {
                   "start": {
                     "line": 8,
-                    "character": 19
```

Comment:
Im assuming this has been checked?

Reply:
outline changes are due to reverting the revert from a previous change, as it is now should be coherent with master (last release) and devel pre-beta3 (one change in beta 3 was missing for this to work)